README.md

```diff
@@ -27,7 +27,8 @@ setup as environment variables.
 | FILTER_API_URL        | http://localhost:22100               | Filter api URL
```

Comment:
If you could update the README.md to specify a need/dependency for the auth-api-stub

Reply:
Added another point to the getting started section about running the stub or zebedee